[2024-05-03T13:39:56.985+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-03T13:39:57.162+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_customers manual__2024-05-03T13:39:42.184579+00:00 [queued]>
[2024-05-03T13:39:57.222+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_customers manual__2024-05-03T13:39:42.184579+00:00 [queued]>
[2024-05-03T13:39:57.234+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-03T13:39:57.316+0000] {taskinstance.py:2327} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_file_customers> on 2024-05-03 13:39:42.184579+00:00
[2024-05-03T13:39:57.415+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=10302) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-03T13:39:57.430+0000] {standard_task_runner.py:63} INFO - Started process 10462 to run task
[2024-05-03T13:39:57.417+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_customers', 'manual__2024-05-03T13:39:42.184579+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpbbeceze4']
[2024-05-03T13:39:57.435+0000] {standard_task_runner.py:91} INFO - Job 93: Subtask upload_file_customers
[2024-05-03T13:39:57.599+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_customers manual__2024-05-03T13:39:42.184579+00:00 [running]> on host 0db60b1edb75
[2024-05-03T13:39:59.119+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_customers' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T13:39:42.184579+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-03T13:39:42.184579+00:00'
[2024-05-03T13:39:59.119+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-03T13:39:59.231+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-03T13:39:59.256+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-03T13:40:00.515+0000] {gcs.py:557} INFO - File /opt/***/dags/data/olist_customers_dataset.csv uploaded to olist_customers_dataset.csv in storage-capstone bucket
[2024-05-03T13:40:00.517+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-03T13:40:00.530+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_customers, execution_date=20240503T133942, start_date=20240503T133957, end_date=20240503T134000
[2024-05-03T13:40:00.630+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-03T13:40:01.416+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-03T13:40:01.445+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
